recipes/gfortran_impl_osx-64/build.sh

```diff
@@ -5,9 +5,8 @@ set -e
 mkdir -p ${SRC_DIR}/build_conda
 cd ${SRC_DIR}/build_conda
 
-export MACOSX_DEPLOYMENT_TARGET="10.9"
-
-env
+export CFLAGS="${CFLAGS} -mmacosx-version-min=${MACOSX_VERSION_MIN}"
+export LDFLAGS="${LDFLAGS} -mmacosx-version-min=${MACOSX_VERSION_MIN}"
```

Comment:
Thanks @isuruf! I found a hard coded value in the build scripts for libgcc at 10.5 as well. I am going to push a patch.

Reply:
The build fails locally with these extra lines. It is the LDFLAGS line. I removed it.

Reply:
The CFLAGS change broke the build too. :(